_posts/2017-09-05-life-at-codurance-robert-firek.md

```diff
@@ -0,0 +1,91 @@
+
+### And what are you currently doing?
+
+I’m currently working with another Software Crafter alongside our client's development team to help improve the legacy code. Once again, this project is completely different from the former once. This team is very experienced in the domain (which is not trivial). Our main goal is to change architecture in the way which will allow reduce the required time to extend and / or introduce features. The company is not based in the UK which also gives the possibility to discover different cultures and food from other regions of Europe ;).
+
+### What do does a typical day look like at Codurance?
+
+Where to begin. There’s no ‘typical’ day at Codurance, since every day is different and a lot depends on the project and the roles which we have in that project. What I can do is point out the most important difference between a ‘typical’ day in Codurance and a typical day in my previous companies.
+
+For me, the biggest difference is the sense of community. Regardless of the projects that we’re working on, the support and knowledge sharing in the team is much stronger than it was in my previous jobs.
+
+We also share and learn ideas outside of our normal project times as well. We organise our own internal sessions after working hours. Say for example that you’re interested in the functional programming or you need to develop your skills in a specific language. You will almost always find more people interested in the same topic. You form a group, book a space in the office and start to study the topic. It is the great way to learn new things and share your knowledge with the others.
+
+
+<blockquote class="hero bq-text-lg"><p>"For me, it’s the sense of community"</p></blockquote>
+
+We also have our bi-weekly catchups on a Tuesday evening, where we meet together on our catch-up meeting to have fun together. We organise a mini-open conference preceded by lightning talks. This is the space where we discuss anything that we want - nothing is off topic. For example, we recently had a few talks about aeronautics. We also have the chance to socialise with people from different projects (which we cannot meet every day). It is the great opportunity for us to build the team and get know each other.
```

Comment:
Biweekly can mean once every two weeks or twice per week. It's a bit ambiguous.
Not sure what else to suggest though, I know 'fortnightly' is not a word many non-native English speakers know.

Reply:
@Gryff Maybe: "Every two weeks, we also have our catchups, ...". It is not important that it is on Tuesday :)